HW03/Vadim_Volovenko/student.rb

```diff
@@ -0,0 +1,27 @@
+class Student
+  attr_reader :name, :surname, :homeworks
+  attr_accessor :mentor
+
+  def initialize(name:, surname:)
+    @name = name
+    @surname = surname
+    @homeworks = []
+    @mentor = []
+  end
+
+  def submit_homework(homework_data)
+    new_homework = Homework.new(homework_data)
+    homeworks << new_homework
+    notificate_mentors(new_homework)
+  end
+
+  def show_homeworks
+    homework.zero? ? 'No homeworks' : homework
```

Comment:
what is `homework` ?
```suggestion
    homeworks.zero? ? 'No homeworks' : homeworks
```